### PR TITLE
project-github-tracker-justine

### PR DESCRIPTION
**Pull request by JustineZwiazek**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500&family=Source+Code+Pro&family=Source+Sans+3&display=swap"
+      rel="stylesheet"
+    />
+    <title>Justine's GitHub Tracker</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div id="profile" class="profile"></div>
+    <img src="images/bag.png" alt="a bag of repos" class="img-bag" />
```
Comment (emmajosefina) [inline code/index.html:19]: LOVE this. Did you create it yourself?

File: code/index.html [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+    <div class="wrapper">
+      <div class="wave"></div>
+      <div id="repoGrid" class="repo-grid"></div>
+      <div class="wave"></div>
```
Comment (emmajosefina) [inline code/index.html:23]: Also this, so creative!

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+// DOM selectors:
+const repoGrid = document.getElementById("repoGrid");
```
Comment (emmajosefina) [inline code/script.js:2]: I like how you name the variables. Easy to read!

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+const profile = document.getElementById("profile");
+
+// Global variables:
+const userName = "JustineZwiazek";
+const API_USER = `https://api.github.com/users/${userName}`;
+const API_REPOS = `https://api.github.com/users/${userName}/repos`;
+
+// Profile information function:
+const getProfile = () => {
```
Comment (emmajosefina) [inline code/script.js:11]: Clean code, mine were a lot messier haha. Great job!

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      <div class="profile-info">
+      <h2>${data.name}</h2>
+      <p>Front End Development student 👩🏻‍💻</p>
+      <p>Stockholm, Sweden 🇸🇪</p></div>
+      `;
+    });
+  getRepos();
+};
+
+// Fetch repositories function:
```
Comment (emmajosefina) [inline code/script.js:26]: Great with the comments

File: code/script.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+          <p>Recent Push: ${new Date(repo.pushed_at).toDateString()}</p>
+          <a href="${repo.html_url}">Link to repo</a>
+          `;
+      });
+      getPullRequests(forkedRepos);
+      drawChart(forkedRepos.length);
+    });
+};
+
+// Fetch pull requests function:
```
Comment (emmajosefina) [inline code/script.js:49]: I had a hard time with this one. Yours was a lot easier to read! Thanks for clearing things up in my head with the comments

File: code/script.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+const getPullRequests = (forkedRepos) => {
+  forkedRepos.forEach((repo) => {
+    fetch(
+      `https://api.github.com/repos/technigo/${repo.name}/pulls?per_page=100`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        const myPullsOnly = data.find(
```
Comment (emmajosefina) [inline code/script.js:57]: Good name for variable

File: code/style.css [hunk 8]
```diff
@@ -1,3 +1,227 @@
-body {
-  background: #FFECE9;
-}
\ No newline at end of file
+* {
```
Comment (emmajosefina) [inline code/style.css:1]: Great with *, I haven't added that in mine, I always use body for that. Thanks!

File: code/style.css [hunk 9]
```diff
@@ -1,3 +1,227 @@
...
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  color: #3a3950;
+}
+
+/* Profile information: */
+.profile {
+  background-color: pink;
```
Comment (emmajosefina) [inline code/style.css:10]: Love this color hehe

File: code/style.css [hunk 10]
```diff
@@ -1,3 +1,227 @@
...
+  background-color: rgb(241, 204, 161);
+}
+
+/* My happy wave: */
+.wave {
+  height: 20px;
+  background-position: 0;
+  background-size: auto 100%;
+  background-repeat: repeat-x;
+  animation: animateCloud 20s linear infinite;
```
Comment (emmajosefina) [inline code/style.css:52]: Such a cool animation! Where did you find the wave-picture?

File: code/style.css [hunk 11]
```diff
@@ -1,3 +1,227 @@
...
+}
+h5 {
+  text-align: center;
+  padding: 20px 0;
+  color: white;
+  text-shadow: 0 0 5px rgb(255, 87, 87);
+}
+
+/* Tablet */
+@media screen and (min-width: 600px) and (max-width: 992px) {
```
Comment (emmajosefina) [inline code/style.css:121]: Nice clean code in here!

File: code/style.css [hunk 12]
```diff
@@ -1,3 +1,227 @@
...
+
+  /* My happy bag */
+  .img-bag-2 {
+    width: 40vw;
+    padding: 40px 0;
+    display: block;
+    object-fit: cover;
+    animation: shake 2s infinite;
+  }
+  @keyframes shake {
```
Comment (emmajosefina) [inline code/style.css:192]: Wow this was new to me!

**Review by emmajosefina — APPROVED** [review]
I adore your page, Justine! It looks very professional. You have made a page that's functional, responsive and fulfills all the requirement for blue level. Also I love your repo bag and animation, really great job! So impressed by your performance this week :)

File: code/index.html (hunk 1 above)
Comment (JustineZwiazek) [reply code/index.html:19]: Ah thanks! I made it in Canva!

File: code/style.css (hunk 10 above)
Comment (JustineZwiazek) [reply code/style.css:52]: Canva again :)

**Comment by JustineZwiazek** [comment]
Thank you for your kind words Emma!

**Pull request closed without merging**